src/java_bytecode/java_types.h

```diff
@@ -18,6 +18,47 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <util/c_types.h>
 #include <util/optional.h>
 
+class java_annotationt : public irept
+{
+public:
+  class valuet : public irept
+  {
+  public:
+    valuet(irep_idt name, exprt value)
+    {
+      id(name);
+      set(ID_value, value);
```

Comment:
Ah yes, agreed, make the value an operand rather than named-operand (vector vs. map allocation).